relay-spans/src/lib.rs

```diff
@@ -6,11 +6,11 @@
     html_favicon_url = "https://raw.githubusercontent.com/getsentry/relay/master/artwork/relay-icon.png"
 )]
 
-pub use crate::span::OtelSpan;
-pub use crate::trace::TracesData;
+pub use crate::span::OtelSpanExt;
+
+pub use opentelemetry_proto::tonic::common::v1 as OtelCommon;
```

Comment:
```suggestion
pub use opentelemetry_proto::tonic::common::v1 as otel_common;
```

It's a module not a type

Reply:
AFAICT `OtelCommon` is only used inside this crate, so I would remove the `pub use` completely.